webhooks.md

```diff
@@ -0,0 +1,105 @@
+# Webhooks
+
+Currently supported webhooks that are triggered everytime when hotel updates the integration.
```

Comment:
This line implies there is a list right after, which has been removed. I'd change it to something like this: 
Partners can be notified about integration changes via webhook calls that are triggered everytime when hotel updates the integration.